### DISCUSSION TITLE
can not download pinia module by "npm i pinia @pinia/nuxt" in nuxt latest version

### DISCUSSION THREAD

**Discussion by lleeeuuuuanchik** [vuejs/pinia#2736] [Help and Questions] [answered]
![image](https://github.com/user-attachments/assets/1388ba59-9892-4cd3-9da8-1edeab83d8d4)
Has anybody resolve this problem?

**Comment by posva** [comment, marked as answer]
In Nuxt, you can also do `npx nuxi@latest module add pinia`
